internal/oci/container.go

```diff
@@ -170,6 +177,8 @@ func (c *Container) StatePath() string {
 
 // CreatedAt returns the container creation time
 func (c *Container) CreatedAt() time.Time {
+	c.opLock.RLock()
```

Comment:
If we make this change, then every call to CreatedAt() that follows a State() call should instead directly access the state.
from what I can see that's in both `server/container_list.go` and `server/container_status.go`

Reply:
Noted.
State() is reverted to current form.

I will keep this in mind if there is follow on PR which changes State() call.